server/adaptors/integrations/__data__/repository/aws_s3/data/sample.json

```diff
@@ -0,0 +1,267 @@
+[
+  {
+    "@timestamp": "2023-07-17T08:14:05.000Z",
+    "body": "078d85edf7268fb4814b1b4fc9f4c64adfde218b6b489a38ecf1b269f14f3c7a centralizedlogging-webconsoleuis3bucket22191f5e-t9bxwwr3d7k [31/Jan/2023:09:25:20 +0000] 35.89.52.162 arn:aws:sts::347283850106:assumed-role/CentralizedLogging-CustomCDKBucketDeployment8693BB-1X4DVR38SF7ZY/CentralizedLogging-CustomCDKBucketDeployment8693BB-kU6BAxSswmfp HQ37919R28X8MPJV REST.GET.BUCKET - \"GET /?list-type=2&prefix=&encoding-type=url HTTP/1.1\" 200 - 322 - 30 29 \"-\" \"aws-cli/1.25.70 Python/3.9.13 Linux/4.14.255-296-236.539.amzn2.x86_64 exec-env/AWS_Lambda_python3.9 botocore/1.27.69\" - ",
+    "event": {
+      "result": "ACCEPT",
+      "name": "s3_log",
+      "domain": "s3.log"
+    },
+    "attributes": {
+      "data_stream": {
+        "dataset": "s3.log",
+        "namespace": "production",
+        "type": "logs"
+      }
+    },
+    "cloud": {
+      "provider": "aws",
+      "account": {
+        "id": "111111111111"
+      },
+      "region": "ap-southeast-2",
+      "resource_id": "s3-centralizedlogging-webconsoleuis3bucket22191f5e-t9bxwwr3d7k",
+      "platform": "aws_s3"
+    },
+    "aws": {
+      "s3": {
+        "bucket_owner": "078d85edf7268fb4814b1b4fc9f4c64adfde218b6b489a38ecf1b269f14f3c7a",
+        "bucket": "centralizedlogging-webconsoleuis3bucket22191f5e-t9bxwwr3d7k",
+        "time": "31/Jan/2023:09:25:20 +0000",
```

Comment:
`aws.s3.time` isn't present in any of the schemas

Reply:
will update this